mtbf_driver/tests/mtbf/keyboard/test_mtbf_keyboard.py

```diff
@@ -4,7 +4,8 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 from mtbf_driver.MtbfTestCase import GaiaMtbfTestCase
-from gaiatest.apps.contacts.app import Contacts
+#from gaiatest.apps.contacts.app import Contacts
```

Comment:
I'll remove it... I comment this line at very beginning...but I forgot to remove it at the end...